watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Sites/SiteFragment.java

```diff
@@ -94,19 +103,22 @@ public void onResume() {
         mParentActivity.setMenuAction(false);
     }
 
-    private void setButtonListeners(View view){
-        View editButton = view.findViewById(R.id.site_edit_site);
+    private void setButtonListeners(View view) {
+        mMenu = (FloatingActionsMenu) view.findViewById(R.id.site_settings);
+        FloatingActionButton editButton = (FloatingActionButton) view.findViewById(R.id.site_edit_site);
         editButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
+                mMenu.collapse();
                 editSite();
             }
         });
-        View miniSiteCreate = view.findViewById(R.id.site_add_minisite);
+        FloatingActionButton miniSiteCreate = (FloatingActionButton) view.findViewById(R.id.site_add_minisite);
```

Comment:
I think we were having back compatibility issues with this before. Can you verify?